app/components/validatable_summary_list_component.rb

```diff
@@ -0,0 +1,21 @@
+class ValidatableSummaryListComponent < GovukComponent::SummaryListComponent
+  registered_slots.delete(:rows)
```

Comment:
Checking I understand what this does: it removes the rows defined in the superclass so we can write our own?

Reply:
That's correct, yes.  Unfortunately there's no "good" way to do that.